engine/sendmessage.go

```diff
@@ -168,8 +171,29 @@ func (p *Engine) sendMessage(ctx context.Context, msg *message.Message) (*notifi
 			ScheduleID:   msg.ScheduleID,
 			Users:        onCallUsers,
 		}
+	case notification.MessageTypeSignalMessage:
+		id, err := uuid.Parse(msg.ID)
+		if err != nil {
+			return nil, errors.Wrap(err, "parse signal message id")
+		}
+		rawParams, err := gadb.New(p.b.db).EngineGetSignalParams(ctx, uuid.NullUUID{Valid: true, UUID: id})
```

Comment:
had to go up & expand almost 200 lines to see that `p` = `Engine`. Still not sure what the `b` is 😬

Reply:
Yes, the engine code badly needs refactor and to be broken apart, the receiver methods are far too long.

IIRC the `b` is just a bunch of prepared statements, a small PR converting them to sqlc is likely in order too.